dapps/onboarding.vue/src/i18n/de.json

```diff
@@ -122,7 +122,7 @@
       "create-profile": {
         "company": {
           "status-1": "Initialisiere Verbindung zum Netzwerk...",
-          "status-2": "Erstellung des Unternehmensaccounts...",
+          "status-2": "Erstellung der Unternehmenidentität...",
```

Comment:
`...` should be always preceded by a space, when it completes a sentences. Maybe you can update that also for the other strings.